roles/monitoring_sap/tasks/main.yml

```diff
@@ -0,0 +1,63 @@
+---
+# 2 blocks executing 2 different actions, controlled by the customer variable  sap_monitoring_action (add/delete)
+
+##########################################
+#  Block: Delete a SAP System from the monitoring configuration
+##########################################
+
+- name: Delete a SAP System from the monitoring configuration
+  when: sap_monitoring_action == "delete"
+  block:
+
+    - name: Check existence of the variables defined in ./vars/sample-monitoring-sap-parameters.yml
+      fail:
+        msg: 'variable {{ sap_monitoring_nr }} not defined in the file ./vars/sample-monitoring-sap-parameters.yml '
+      when: sap_monitoring_nr is not defined
+
+    - name: Delete one SAP System from the monitoring configuration
+      ansible.builtin.include_tasks: sap-monitoring-configuration-deletion.yml
+
+
+#######################################
+#  Block: Add a SAP System to the monitoring configuration
+#######################################
+
+- name: Add a SAP System to the monitoring configuration
+  when: sap_monitoring_action == "add"
+  block:
+
+    - name: Check OS architecture
+      fail :
+        msg: 'Required architecture is x86_64'
+      when: ansible_architecture != "x86_64"
+      
+    - name: Check distribution version architecture
```

Comment:
Check OS distribution

Reply:
the relase version check is  the next check. Separated because of the different return message.